### PR TITLE
Extract common labels to _helpers.tpl

### PR DESCRIPTION
**Pull request by amirschw**
**Reason for Change**:
The common labels are used in all templates throughout the chart. This PR extracts these labels to the `_helpers.tpl` file to make them reusable.

**Issue Fixed**:

**Notes for Reviewers**:
Wasn't sure whether I should update the chart version in the `Chart.yaml`  as it's the same as the appVersion that hasn't been changed.

### CODE REVIEW

File: charts/aad-pod-identity/templates/mic-deployment.yaml [hunk 1]
```diff
@@ -3,10 +3,7 @@ kind: Deployment
 metadata:
   name: {{ template "aad-pod-identity.mic.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ template "aad-pod-identity.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ template "aad-pod-identity.chart" . }}
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
```
Comment (aramase) [inline charts/aad-pod-identity/templates/mic-deployment.yaml]: These same labels are used in the [selector.matchLabels](https://github.com/Azure/aad-pod-identity/blob/master/charts/aad-pod-identity/templates/mic-deployment.yaml#L15-L19). Can you please update that too in both the mic-deployment and nmi-daemonset?

Comment (amirschw) [reply charts/aad-pod-identity/templates/mic-deployment.yaml]: Done

**Pull request merged**